### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivan-shpynda**
[DEMO LINK](https://ivan-shpynda.github.io/react_sum/)

### CODE REVIEW

**Review by Taras-Konovalov — APPROVED** [review]
https://prnt.sc/132u3o2

Good job!

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1>React sum</h1>
+    <Sum
+      a={2}
+      b={2}
+    />
```
Comment (VinnieJ-2k20) [inline src/App.js:11]: No need to add extra lines here, code is short enough.
```suggestion
    <Sum a={2} b={2} />
```

File: src/components/Sum.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <p>
+      Sum of
+      {a}
+      {' '}
+      and
+      {b}
+      {' '}
+      is
+      {a + b}
```
Comment (VinnieJ-2k20) [inline src/components/Sum.js:14]: There is much easier way to render this text.
Write it in one line using curly braces and backquotes:
```suggestion
{`Sum of ${a} and ${b} is ${a + b}`}
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
You forgot to add spaces between words and numbers:
image
Some minor edits also needed.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**